resharper/resharper-unity/src/CSharp/Daemon/Stages/PerformanceCriticalCodeAnalysis/PerformanceCriticalCodeStageUtil.cs

```diff
@@ -135,6 +135,25 @@ public static bool IsNullComparisonWithUnityObject(IEqualityExpression equalityE
             return false;
         }
 
+        public static bool HasPerformanceSensitiveAttribute(IFunctionDeclaration functionDeclaration)
+        {
+            return HasSpecificAttribute(functionDeclaration, "PerformanceCharacteristicsHintAttribute");
```

Comment:
I think this name only works if we're giving it an argument that specifies what the characteristics are. I'll add a top level comment about attribute names